src/rtcmix/iosstuff/IOSLIBOBJS

```diff
@@ -35,8 +35,9 @@ LIBGENOBJS = ../../genlib/allpass.o ../../genlib/allpole.o \
 LIBINLETOBJS = ../control/maxmsp/RTInletPField.o ../control/maxmsp/inletglue.o
 LIBPFBUSOBJS = ../control/pfbus/PFBusPField.o ../control/pfbus/pfbusglue.o
 LIBMINCOBJS = ../parser/minc/Node.o ../parser/minc/callextfunc.o \
-../parser/minc/minc.o ../parser/minc/error.o ../parser/minc/sym.o \
```

Comment:
Hah!  Good catch -- but are you actually building for iOS?  This stuff is so out of date!

Reply:
No, definitely not. However, I am doing static builds, and `IOSLIBOBJS` is included by `src/rtcmix/Makefile` for all static builds:
https://github.com/RTcmix/RTcmix/blob/6ad551a8930b1a0b3304b10dee9e6a99bab5a659/src/rtcmix/Makefile#L209-L212
Maybe this file should be moved/renamed?